core/update.php

```diff
@@ -300,6 +300,10 @@ function update_info_page() {
   if (db_table_exists('cache_update')) {
     cache('update')->flush();
   }
+  // Flush the theme cache so we can render this page correctly if themes have
+  // been updated.
```

Comment:
Small rewording of the code comment:

```suggestion

  // Flush the theme cache so we can render this page correctly if the theme
  // registry been updated with new preprocess or template variables. 
```